.ci/Jenkinsfile

```diff
@@ -55,7 +55,7 @@ pipeline {
         axes {
           axis {
             name 'PLATFORM'
-            values 'macosx && x86_64', 'ubuntu-20.04 && immutable', 'windows-2019 && windows-immutable'
+            values 'ubuntu-20.04 && immutable', 'windows-2019 && windows-immutable'
```

Comment:
macos is not fully supported, recommended to use GitHub actions instead, i'll raise a PR with the proposal